.travis.yml

```diff
@@ -12,7 +12,7 @@ install:
     - "pip install selenium==2.53.0"
     - "pip uninstall -y xblock-drag-and-drop-v2"
     - "python setup.py sdist"
-    - "pip install dist/xblock-drag-and-drop-v2-2.0.16.tar.gz"
+    - "pip install dist/xblock-drag-and-drop-v2-2.0.17.tar.gz"
```

Comment:
Note: This will removed before merge and a new Release PR will be created with this.